third_party/amd/python/triton_amd.cc

```diff
@@ -217,15 +217,14 @@ void init_triton_amd(py::module &&m) {
     for (llvm::Function &f : module->functions()) {
       if (f.hasExternalLinkage() && f.hasName() && !f.hasExactDefinition()) {
         llvm::StringRef funcName = f.getName();
-        // Rules for linking the extern lib:
-        // 1. if __nv_ is found in the module, we'll link all four libs:
-        //    cuda2gcn, opencl, ocml, and ockl. Note that opencl might
-        //    not be needed. But we add it here and will try to remove
-        //    it in the future.
-        // 2. if the function name includes ocml or ockl, only link
+        // The rule for linking the extern lib:
+        //    if the function name includes ocml or ockl, link
         //    ocml or ockl accordingly.
-        if (funcName.contains(lib) || funcName.contains("__nv_"))
+        if (funcName.contains(lib))
           return true;
+        if (funcName.contains("__nv_"))
+          throw std::runtime_error(
+              "implicit convertsion of `__nv_*` instructions has been dropped");
```

Comment:
Also mention what the developers should do. Like something `implicit conversion of CUDA __nv_<op> intrinsics has been dropped; please update your source program to use triton.language.extra.<op> to replace triton.language.extra.cuda.<op>`. This way developers can immediately act on it.

Reply:
Thanks @antiagainst. Done!